### PR TITLE
Switch from Test.srcdir to Test.workdir

### PR DESCRIPTION
**Pull request by clebergnu**
Due to Test.srcdir deprecation on Avocado.

Reference: https://github.com/avocado-framework/avocado/pull/2509
Signed-off-by: Cleber Rosa <crosa@redhat.com>

---

Changes from v1 (#817):
 * Fixed typo (`s/wordir/workdir`)
 * Give unique directory for filebench installation (`--prefix=`) in `fs/filebench.py`

### CODE REVIEW

File: io/disk/lvsetup.py [hunk 1]
```diff
@@ -81,7 +81,7 @@ def setUp(self):
         self.lv_name = lv_name
         if lv_utils.lv_check(vg_name, lv_snapshot_name):
             self.cancel('Snapshot %s already exists' % lv_snapshot_name)
-        self.mount_loc = self.srcdir
+        self.mount_loc = self.workdir
```
Comment (harish-24) [inline io/disk/lvsetup.py]: https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/io/disk/lvsetup.py#L72 uses `workdir` already. With this change, the test would try to mount on the `workdir` which is used as `ramdisk_basedir`.

Comment (narasimhan-v) [reply io/disk/lvsetup.py]: @clebergnu can this small change be taken care of with latest changes being pulled ?

**Comment by clebergnu** [comment]
Sure, I'm mostly offline today, but I'll get that done ASAP.

**Comment by PraveenPenguin** [comment]
@clebergnu can you please have a look on this PR

**Comment by clebergnu** [comment]
@PraveenPenguin working on it.

**Comment by clebergnu** [comment]
@PraveenPenguin @narasimhan-v sent as v3 (#887)

**Pull request closed without merging**